docs/_includes/sidebar-nav.html

```diff
@@ -1,22 +1,22 @@
-<div class="flex-shrink-0 p-3 bg-white sticky-top" id="detail-sidebar">
```

Comment:
I removed the `p-3` because it seemed like we didn't need it but can add it back :) the id isn't being used anymore so I removed that.